src/pages/audio-rewards-page/ChallengeRewardsTile.tsx

```diff
@@ -133,15 +148,35 @@ const useRewardIds = () => {
   return filteredRewards
 }
 
+const CHALLENGE_REFRESH_INTERVAL_MS = 5000
```

Comment:
matches the DN challenge task interval, but do we want it this often?

Reply:
seems like a good candidate for remote config. seems like a sensible starting value though